src/components/tabs/sell-tab-content/sell.process.tsx

```diff
@@ -171,7 +158,7 @@ export function SellTabContentProcess({ asset, balance }: SellTabContentProcessP
     if (!validatedData || !validatedData.amount || !validatedData.asset || !address || !paymentInfo) return;
     setIsCompleting(true);
     await updateBankAccount();
-    // TODO: #LN-ALBY# create transaction
+    sendPayment(paymentInfo.paymentRequest);
```

Comment:
Isn't there something missing here? Something like
```ts
      .then((txId) => setSellTxId(txId))
      .finally(() => setIsCompleting(false));
```

Reply:
yes you are right. but we don't receive any txId of webln, I will check again how it is working out with the await and the UI